bzt/modules/services.py

```diff
@@ -51,3 +55,70 @@ def prepare(self):
             unpacked_list.append(archive[:-4])  # TODO: replace with top-level archive content
 
         replace_in_config(self.engine.config, packed_list, unpacked_list, log=self.log)
+
+
+class Recorder(Service):
+    def __init__(self):
+        super(Recorder, self).__init__()
+        self.proxy = None
+        self.headers = {}
+        self.base_url = 'https://a.blazemeter.com/api/latest/mitmproxies'
+
+    def prepare(self):
+        super(Recorder, self).prepare()
+        token = self.parameters.get('token', ValueError("You need token for recording"))
+        self.headers = {"X-Api-Key": token}
+        self.log.debug('Create Blazemeter proxy')
+        while True:
+            req = requests.delete(self.base_url, headers=self.headers)
```

Comment:
if there is existing proxy, let's not delete it, just re-use it. stop it if it is already started and clear the results. might be important to not re-generate SSL certs